packages/libs/eda/src/lib/core/api/DataClient/index.ts

```diff
@@ -57,7 +59,17 @@ export default class DataClient extends FetchClientWithCredentials {
         method: 'POST',
         path: `/apps/${computationName}/visualizations/${visualizationName}`,
         body: params,
-        transformResponse: ioTransformer(decoder),
+        transformResponse(body) {
+          if (body == null) {
+            throw new NoDataError(
+              'The visualization cannot be made because the current subset is empty.',
```

Comment:
Probably the first of the suggestions below.
In the map we don't want to say "subset" if poss.
There's still an issue that the user might think "but I haven't added any filters" - some of these 204 no datas can be caused by little filters.
I think we'll have to punt that one to the next release. I'm wondering if the filter chips area in the header should show read-only chips for the viewport, timeslider, marker-config and selected markers?

```suggestion
              'The visualization cannot be made because no data remains after filtering.',
```

```suggestion
              'The visualization cannot be made because all data has been filtered out.',
```